web-client/src/app/components/pay-amount-form/pay-amount-form.component.html

```diff
@@ -0,0 +1,38 @@
+<form [formGroup]="paymentForm" (ngSubmit)="onSubmit()">
+  <div class="ion-text-center">
+    <ion-input
+      formControlName="amount"
+      name="amount"
+      type="text"
+      inputmode="decimal"
+      placeholder="Amount"
+      [autofocus]="autofocus"
+      [ngClass]="{ invalid: amountErrors }"
+      class="text-2xl font-audiowide"
+    ></ion-input>
+
+    <ion-text color="danger" *ngIf="amountErrors">
+      <div *ngIf="amountErrors.required">Required</div>
+      <!-- Skip the additional numeric message if it's just empty. -->
+      <div *ngIf="amountErrors.numeric && !amountErrors.required">
+        Please enter a number
+      </div>
+      <div *ngIf="amountErrors.min">
+        Amount too low (minimum {{ amountErrors.min.min }})
```

Comment:
Perhaps this line of code could say "Insufficient amount in your balance"

Reply:
Yeah, we can improve this. I think we should tackle this in a follow-up pass, though.